src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,63 @@
 import React from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  setMovies: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ setMovies }) => {
+  const [query, setQuery] = React.useState('');
+  const [movie, setMovie] = React.useState<Movie | null>(null);
+  const [isLoading, setLoading] = React.useState(false);
+  const [isError, setError] = React.useState(false);
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    setLoading(true);
+
+    const movieFromApi = await getMovie(query);
+
+    setLoading(false);
+
+    if ('Error' in movieFromApi) {
+      setError(true);
+
+      return;
+    }
+
+    const {
+      Title, Poster, imdbID, Plot,
+    } = movieFromApi;
+
+    setMovie({
+      title: Title,
+      description: Plot,
+      imgUrl:
+        Poster !== 'N/A'
+          ? Poster
+          : 'https://via.placeholder.com/360x270.png?text=no%20preview',
+      imdbUrl: `https://www.imdb.com/title/${imdbID}`,
+      imdbId: imdbID,
+    });
+  };
+
+  const handleAdd = () => {
+    if (movie) {
```

Comment:
you already handle it
![image](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/91411692/d22e793b-04d5-438a-8920-392e0e3d5a1a)

```suggestion
```